ipatests/test_integration/test_acme.py

```diff
@@ -0,0 +1,132 @@
+#
+# Copyright (C) 2020  FreeIPA Contributors see COPYING for license
+#
+
+import time
+
+import pytest
+
+from ipalib.constants import IPA_CA_RECORD
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+from ipaplatform.osinfo import osinfo
+
+
+# RHEL does not have certbot.  EPEL's version is broken with
+# python-cryptography-2.3; likewise recent PyPI releases.
+# So for now, on RHEL we suppress tests that use certbot.
+skip_certbot_tests = osinfo.id not in ['fedora',]
+
+
+class TestACME(IntegrationTest):
+    """
+    Test the FreeIPA ACME service by using ACME clients on a FreeIPA client.
+
+    Right now the only thing we test is the Certbot client using
+    http-01 challenge with Certbot's standalone HTTP server.
+    We can add tests for DNS challenges later.
```

Comment:
Do you want to update this comment to include something about mod_md?

Reply:
This comment was updated in a later commit.